dev-portal/public/custom-content/content-fragments/GettingStarted.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Getting Started
+---
+
+## Getting Started
+
+### Create an account and subscribe to APIs
+
+To use any of our APIs you must have a developer account. A developer account gives you an API Key, access to use and test our APIs, and metrics on your usage. Create or sign in using the buttons in the top right.
+
+When you first register, your new API Key won't be tied to any of our APIs. To activate it for an API, navigate to [APIs](/apis), then find the API you want to use and click `subscribe`. Your API Key is now subscribed to the API and you can make calls to all methods on the API.
+
+Figuring out how to use APIs can be hard. Use the “Try it out!” feature to get examples of the request and response shapes of our APIs. The Developer Portal will make an API call to the backend service using your API key and shows a sample `curl` request with all necessary input parameters and the real response.
+
+If you need your API Key for any reason, you can always find it on [your dashboard](/dashboard) after logging in.
+
```

Comment:
Updated text slightly
-----------------------
To use any of our APIs you must create a developer account. A developer account provides an API Key for accessing our APIs, a playground for testing our APIs, and provides API usage metrics. Create or sign in using the buttons in the top right.

After you create a new account, you will have a new API Key but it won't be linked to any of our APIs. To activate it for a particular API, navigate to [APIs](/apis) and find the API you want. Click `subscribe`. Your API Key is now subscribed to the API and you can make calls to its methods.

We know that figuring out how to use APIs can be hard. Use the “Try it out!” feature to get examples of the request and response shapes of our APIs. This makes an API call to the backend service using your API key and provides a sample `curl` request with all necessary input parameters and the real response.

If you need your API Key for any reason, you can always find it on [your dashboard](/dashboard) after logging in.

Reply:
This text is good, but it doesn't set expectations for subscribing to multiple api-stages (because they're on the same usage plan). Is this the right place to set that expectation?

If so, perhaps something like:
```
Depending on the API owners' configuration, subscribing to one API may subscribe you to several APIs. These APIs may have related functionality or share a pricing plan, and requests against any of them will be counted together in your usage.
```

Reply:
I like it. @a-tan -- thoughts?

Reply:
I would remove the "API owners'" part because its written from the perspective of the API owner
`Depending on the API configuration, subscribing to one API may subscribe you to several APIs. These APIs may have related functionality or share a pricing plan. Requests against any of them will be counted together in your usage.`